lua/jc/server.lua

```diff
@@ -2,7 +2,11 @@ local Job = require("jc.jobs")
 local M = {}
 local data_dir = vim.fn["project_root#get_basedir"]("data")
 local vendor_dir = vim.fn["project_root#get_basedir"]("vendor")
-local project_name = vim.fn.substitute(vim.fn["project_root#find"](), "[\\/:;.]", "_", "g")
+local project_root_file = vim.fn["project_root#find"]()
+if vim.fn.filereadable(project_root_file) == 1 then
+    vim.cmd('lcd ' .. vim.fn.fnamemodify(project_root_file, ':h'))
+end
+local project_name = vim.fn.substitute(project_root_file, "[\\/:;.]", "_", "g")
```

Comment:
We need to evaluate this variables when java file opened. Because if we open vim not in project folder we will have inconsistent in this variables.